### PR TITLE
Newsletter 13: Egregoria

### PR DESCRIPTION
**Pull request by Uriopass**
Part of #232

### CODE REVIEW

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -62,6 +62,29 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Egregoria]
+
+![Egregoria buildings screenshot](egregoria.png)
+
+Egregoria was previously known as Scale,
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:69]:
```suggestion
Egregoria was previously known as Scale,
```

File: content/posts/newsletter-013/index.md [hunk 2]
```diff
@@ -62,6 +62,29 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+[Egregoria]'s objective is to become a granular society simulation,
+filled with fully autonomous agents interacting with their world in real time.  
+
+The 5th [devlog][egregoria-blog-post] was published, talking about
+the renaming, project managment, buildings and scripting.  
+
+A [Discord][egregoria-discord] server was launched to discuss the project.  
+
+_Discussions:  
+[/r/rust_gamedev](https://www.reddit.com/r/rust_gamedev/comments/igzbl9/egregoria_devblog_5/)
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:81]:
```suggestion
[/r/rust_gamedev](https://www.reddit.com/r/rust_gamedev/comments/igzbl9/egregoria_devblog_5/)
```

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
Looks good, but a few bits of trailing whitespace are tripping up the linter:

**Comment by Uriopass** [comment]
Done ✔️

**Review by 17cupsofcoffee — APPROVED** [review]

**Review by ozkriff — APPROVED** [review]
:+1: (made a few small style/fmt tweaks for consistency)

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**